packages/typespec-ts/src/transform/transformTelemetryInfo.ts

```diff
@@ -37,7 +40,7 @@ function getCustomRequestHeaderNameForClient(
     );
     for (const op of operations) {
       const headerName = getCustomRequestHeaderNameForOperation(
-        ignoreDiagnostics(getHttpOperation(program, op))
+        ignoreDiagnostics(getHttpOperationWithCache(program, op))
```

Comment:
Can you point to me where is getCustomRequestHeaderNameForClient being called ? thx.